chess-AI/train.py

```diff
@@ -1,32 +1,46 @@
-from functools import reduce
-
-import chess
+"""
+Main training program
+"""
+import time
 import os
+import chess
 import torch
-import numpy as np
+
 from torch.utils.data import DataLoader, TensorDataset
```

Comment:
`chess` and `torch` are not built-ins, so they should be grouped with the other torch include.
```py
import os
import time

import chess
import torch
from torch.utils.data import DataLoader, TensorDataset
```